src/ci/bin/test.inc.sh

```diff
@@ -373,10 +375,8 @@ cromwell::private::create_build_variables() {
     CROMWELL_BUILD_DOCKER_TAG="${CROMWELL_BUILD_DOCKER_TAG:0:128}"
     CROMWELL_BUILD_DOCKER_TAG="${CROMWELL_BUILD_DOCKER_TAG//[^a-zA-Z0-9.-]/_}"
 
-    CROMWELL_BUILD_OPTIONAL_SECURE="${CROMWELL_BUILD_OPTIONAL_SECURE-false}"
```

Comment:
Why do we not need this anymore?

Reply:
In the past, we had the ability to _try_ and login for some builds. Especially the build called `sbt`.

- For internal PRs, the `sbt` build would log in and publish artifacts to dockerhub/jfrog.
- For external PRs, the `sbt` build would detect that vault info wasn't available, and would just run the tests, and skip publishing artifacts.

Basically it was a way that we could still run external PRs and enhance internal PRs.

Now thanks to docker.io's [pull limits](https://www.docker.com/increase-rate-limits) as of a few months ago, we _always_ want _all_ internal PRs to be enhanced by logging in to docker.io.